### ISSUE TITLE
Creating elements on page load on index.js

### ISSUE THREAD

**Issue by jwld** [facn2/facnrecipe#54]
https://github.com/facn2/facnrecipe/blob/cc48ffcd72c4f6e2e992a57ad7fe2047f067b46a/views/index.js#L15-L43

This should all just be hardcoded into the html file

**Comment by tyeung2017** [comment]
notes taken. Was in a hurry . haha

**Comment by ghost** [comment]
Absolutely. We're realizing now why that's a bad idea. Dealing with the CSS became really complicated towards the end because of this.
